config.toml

```diff
@@ -176,7 +176,7 @@ enable = false
   url = "https://dell.github.io/csm-docs/docs/"
 
 [[params.versions]]
-  version = "v1.10.0"
+  version = "v1.10.2"
```

Comment:
Should it be `v1.11.0` ? Please check.

Reply:
This is the previous version the current version is already 1.11.0: https://github.com/dell/csm-docs/blob/706d4654414eeb90d9047d03b8e27cc955982293/config.toml#L175